README.md

```diff
@@ -1,2 +1,46 @@
 # TDD-project
 Practicing test driven development
+
+## Getting Started
+
+### Setup
+```
+git clone  <https://github.com/deyemiobaa/tdd-project.git>
+```
+- run ```cd tdd-project``` to enter the project folder
+- run `bundle install`
+- run `ruby main.rb` to launch console app
```

Comment:
[OPTIONAL] There is no `main.rb` file in your project. It would be better to remove line 12 from your Readme.
Since you mentioned the bundle install command in your Readme, I would recommend adding `gem 'rspec'` to Gemfile so that the user is able to install RSpec and test your project by running the `bundle install` command.